cli/fathom_web/vectorizer.py

```diff
@@ -175,7 +174,11 @@ def fathom_fox_addon(ruleset_file):
                 copyfileobj(ruleset_file, new_ruleset_file)
 
             # Build FathomFox:
-            run(fathom_fox / 'node_modules' / '.bin' / 'rollup',
+            if platform.system() == 'Windows':
```

Comment:
We have this "cmd-or-not" block repeat a few times. It might be a good thing to factor up.